webpack/assets/javascripts/react_app/components/common/EmptyState/DefaultEmptyState.js

```diff
@@ -14,19 +15,21 @@ const DefaultEmptyState = props => {
     documentation: {
       url,
       label = __('For more information please see'),
-      buttonLabel = __('Documentation'),
+      buttonLabel = __('documentation'),
```

Comment:
oh, why did you rename it? :)

Reply:
I thought we should get rid of the capital letter cause, it had a capital word right in the middle of the sentence.
```
For more information please see Documentation
```